README.md

```diff
@@ -83,6 +83,12 @@ require('noirbuddy').setup {
 
 ![](presets.png)
 
+> [!CAUTION]
+> When using lazy.nvim, you **must** use the `install = { colorscheme = { "noirbuddy" }
```

Comment:
I think "must" could be rephrased a bit since I don't have this setting yet everything works just fine. Something like "When using lazy.nvim, if you're facing issues with preset you might want to set X". WDYT?

Reply:
Yeah, that seems like a reasonable rewording. FWIW I think the reason it loads fine without setting either the `install = ...` or `vim.opt.colorscheme` is you don't lazy load nvim-noirbuddy and it "activates" itself on load. I *believe* this may be the source of some bad behavior when multiple colorscheme plugins are installed and loaded. It might be better to have `lazy = true` and let lazy.nvim activate it based on the above configuration. This is how my setup works and it means I don't get strange behavior when I switch colorschemes.

Reply:
Merged as is! Thanks again! <3

Reply:
Oh! I was still going to reword it when I had the chance :sweat_smile: